tests/vppoutputencode.h

```diff
@@ -23,17 +23,48 @@
 #define vppoutputencode_h
 #include "VideoEncoderHost.h"
 #include "encodeinput.h"
+#include <string>
 #include <vector>
 
 #include "vppinputoutput.h"
+using std::string;
+//#include "yamitranscodehelp.h"
 
+class EncodeParams
+{
+public:
+    EncodeParams();
+
+    VideoRateControl rcMode;
+    int32_t initQp;
+    int32_t bitRate;
+    int32_t fps;
+    int32_t ipPeriod;
+    int32_t ipbMode;
+    int32_t kIPeriod;
+    uint32_t fourcc;
```

Comment:
fourcc sets vpp output fomat, and doesn't set encoding input format in some cases, right? If yes, better move fourcc to CmdLineParams